src/dllist.c

```diff
@@ -1520,6 +1520,49 @@ static int dllist_set_item(PyObject* self, Py_ssize_t index, PyObject* val)
     return 0;
 }
 
+
+static PyObject* dllist_index(DLListObject *self, PyObject *value)
+{
+
+    DLListNodeObject *node;
+    Py_ssize_t idx;
+
+    node = (DLListNodeObject *) self->first;
+    idx = 0;
+
+    while ( (PyObject*)node != Py_None )
+    {
+        if( node->value == value )
+            return PyLong_FromSsize_t(idx);
```

Comment:
`PyLong_FromSsize_t` is unavailable in Python 2.5, so we need to drop support for it in README and documentation. Shouldn't be an issue (2.5 was EOLed years ago).